source/SIL.AppBuilder.Portal.Frontend/src/data/containers/resources/reviewer/with-data-actions.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { withData as withOrbit, WithDataProps } from 'react-orbitjs';
+import { ResourceObject } from 'jsonapi-typescript';
+
+import { defaultOptions, REVIEWERS_TYPE } from '@data';
+import { ReviewerAttributes } from '@data/models/reviewer';
+
+
+export interface IProvidedProps {
+  createRecord: (attrs: ReviewerAttributes, relationships) => any;
+  removeRecord: (id: string) => any;
+  updateAttribute: (attribute: string, value: any) => any;
+  updateAttributes: (attrs: ReviewerAttributes) => any;
+}
+
+interface IOwnProps {
+  reviewer: ResourceObject<REVIEWERS_TYPE, ReviewerAttributes>;
+}
+
+type IProps =
+  & IOwnProps
+  & WithDataProps;
+
+export function withDataActions<T>(WrappedComponent) {
+
+  class ReviewerDataActionWrapper extends React.Component<IProps & T> {
+
+    createRecord = async (attributes: ReviewerAttributes, relationships) => {
+
+      const { dataStore } = this.props;
+
+      const c = await dataStore.update(
+        q => q.addRecord({
+          type: 'reviewer',
+          attributes,
+          relationships
+        }),
+        defaultOptions()
+      );
+    }
+
+    removeRecord = async (reviewer) => {
+
+      const { dataStore } = this.props;
+
+      const r = await dataStore.update(
+        q => q.removeRecord({
```

Comment:
does this work if you just pass reviewer? (less code is best code :) )

Reply:
it does!! changed it